modules/ROOT/pages/branded_android_app/update_android_app_2021.adoc

```diff
@@ -0,0 +1,152 @@
+* `Alias` is the `app name` (here: damkencloud) aka the `Key Alias`.
+
+
+[source,java]
+----
+java -jar pepk.jar --keystore=foo.keystore --alias=foo --output=encrypted_private_key_path
+----
+
+image:branded_android_app/android_app_build_17.png[image]
+
+After executing the above command, you will be prompted to enter both the passwords for "your keystore password" and then for "your key alias password".
+Keep in mind, that these passwords are invisible when typing them. (Multiple entries or typos lead to errors.)
+
+You can also check your signing parameters in ownBrander, if you have uploaded them for previous versions.
+
+image:branded_android_app/android_app_build_18.png[image]
+
+In case you downlaod the PEPK tool on macOS, you'll need additional permissions in the macOS "Security & Privacy" settings:
+
+image:branded_android_app/android_app_build_19.png[image]
+
+Proceed by clicking on the button to upload your private key
+
+image:branded_android_app/android_app_build_20.png[image]
+
+Double-click the file to upload it. After a successful upload, click on the "save" button (bottom right of the page).
+
+The following is a depiction of the upload file.
+
+image:branded_android_app/android_app_build_21.png[image]
+
+You will be redirected to the "Terms of Use" page. Click "Accept" after reading them.
+
+Navigate back to "Production". The certificate is now displayed.
+
+image:branded_android_app/android_app_build_22.png[image]
+
+In the "Production" bar move to the release review button.
+
+image:branded_android_app/android_app_build_23.png[image]
+
+Notice the check-mark by the "App Integrity" field.
+
+image:branded_android_app/android_app_build_24.png[image]
+
+== Upload Android App Bundle
+
+Proceed by clicking on "upload" in order to upload the `*-release.aab` file you previously downloaded from your shared account (personal folder) on customer.owncloud.com.
+
+image:branded_android_app/android_app_build_25.png[image]
+
+image:branded_android_app/android_app_build_26.png[image]
+
+You should see Googles colors changing during the upload process. Thereafter, a preview of the app release candidate.
+
+image:branded_android_app/android_app_build_27.png[image]
+
+The new version is available and should be thoroughly examined before releasing it to the production environment.
+
+image:branded_android_app/android_app_build_28.png[image]
+
+Optionally, you may choose to provide your users with information regarding the changelog, so they know which changes have been implemented.
+
+image:branded_android_app/android_app_build_29.png[image]
+
+If available from a previous release, just copy it.
+
+image:branded_android_app/android_app_build_30.png[image]
+
+After saving any modifications, proceed by clicking the "check release" button.
+
+image:branded_android_app/android_app_build_31.png[image]
+
+You may see some warnings. (If of importance, check to see in what regards they are.) Scroll down.
```

Comment:
Is there a color scheme or something that helps users decide what might be important?

Reply:
The warnings (see screenshot) appear as yellow triangles. I feel that that is self-explanatory.